inventory-service/src/main/java/com/petclinic/inventoryservice/presentationlayer/InventoryController.java

```diff
@@ -128,5 +128,10 @@ public Mono<ResponseEntity<InventoryTypeResponseDTO>> addInventoryType(@RequestB
                 .defaultIfEmpty(ResponseEntity.notFound().build());
     }
 
+    @GetMapping("/type")
```

Comment:
The GetMapping isn't off? Shouldn't it be /inventory/type?

Reply:
Nice observation, but the /inventory should already be included in the @RequestMapping at the beggining, adding /inventory/type would simply be redundant no?